frontend/packages/dev-console/src/components/BuildConfigPage.tsx

```diff
@@ -1,30 +1,34 @@
 import * as React from 'react';
 import { Helmet } from 'react-helmet';
 import { match as RMatch } from 'react-router';
+import * as _ from 'lodash';
 import { BuildConfigsPage } from '@console/internal/components/build-config';
+import { withStartGuide } from '../../../../public/components/start-guide';
 import DefaultPage from './DefaultPage';
 
 export interface BuildConfigPageProps {
   match: RMatch<{
     ns?: string;
   }>;
 }
-const BuildConfigPage: React.FC<BuildConfigPageProps> = (props) => {
-  const namespace = props.match.params.ns;
+const allParams = (props) => Object.assign({}, _.get(props, 'match.params'), props);
```

Comment:
I know this is a pattern taken from console but it's not right.
The `noProjectsAvailable` comes in as a prop from `withStartGuide` and never from `match.params`. It would be better to be explicit:
```
const { noProjectsAvailable } = props;
```
Or deconstruct from the params.

Reply:
Done